ui/core/encounter.ts

```diff
@@ -198,6 +196,9 @@ export class Encounter {
 	}
 
 	static updateProtoVersion(proto: EncounterProto) {
+		if (!(proto.apiVersion < CURRENT_API_VERSION)) {
```

Comment:
@NerdEgghead Moved the logic here to not have to duplicate the update logic everywhere you want to apply it with the same if statement.